pyproject.toml

```diff
@@ -1,2 +1,63 @@
-[tool.black]
-line-length = 79
```

Comment:
Let's keep the original setting for `black`, so the code should remain untouched

Reply:
done